test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -451,33 +452,31 @@ public void testDeniedAudits() throws AccumuloSecurityException, AccumuloExcepti
     // We don't want the thrown exceptions to stop our tests, and we are not testing that the
     // Exceptions are thrown.
 
-    try {
-      auditAccumuloClient.tableOperations().create(NEW_TEST_TABLE_NAME);
-    } catch (AccumuloSecurityException ex) {}
-    try {
-      auditAccumuloClient.tableOperations().rename(OLD_TEST_TABLE_NAME, NEW_TEST_TABLE_NAME);
-    } catch (AccumuloSecurityException ex) {}
-    try {
-      auditAccumuloClient.tableOperations().clone(OLD_TEST_TABLE_NAME, NEW_TEST_TABLE_NAME, false,
-          Collections.emptyMap(), Collections.emptySet());
-    } catch (AccumuloSecurityException ex) {}
-    try {
-      auditAccumuloClient.tableOperations().delete(OLD_TEST_TABLE_NAME);
-    } catch (AccumuloSecurityException ex) {}
-    try {
-      auditAccumuloClient.tableOperations().offline(OLD_TEST_TABLE_NAME);
-    } catch (AccumuloSecurityException ex) {}
+    assertThrows(AccumuloSecurityException.class,
+        () -> auditAccumuloClient.tableOperations().create(NEW_TEST_TABLE_NAME));
+
+    assertThrows(AccumuloSecurityException.class, () -> auditAccumuloClient.tableOperations()
+        .rename(OLD_TEST_TABLE_NAME, NEW_TEST_TABLE_NAME));
+
+    assertThrows(AccumuloSecurityException.class,
+        () -> auditAccumuloClient.tableOperations().clone(OLD_TEST_TABLE_NAME, NEW_TEST_TABLE_NAME,
+            false, Collections.emptyMap(), Collections.emptySet()));
+
+    assertThrows(AccumuloSecurityException.class,
+        () -> auditAccumuloClient.tableOperations().delete(OLD_TEST_TABLE_NAME));
+
+    assertThrows(AccumuloSecurityException.class,
+        () -> auditAccumuloClient.tableOperations().offline(OLD_TEST_TABLE_NAME));
+
     try (Scanner scanner = auditAccumuloClient.createScanner(OLD_TEST_TABLE_NAME, auths)) {
-      scanner.iterator().next().getKey();
-    } catch (RuntimeException ex) {}
-    try {
-      auditAccumuloClient.tableOperations().deleteRows(OLD_TEST_TABLE_NAME, new Text("myRow"),
-          new Text("myRow~"));
-    } catch (AccumuloSecurityException ex) {}
-    try {
-      auditAccumuloClient.tableOperations().flush(OLD_TEST_TABLE_NAME, new Text("myRow"),
-          new Text("myRow~"), false);
-    } catch (AccumuloSecurityException ex) {}
+      assertThrows(RuntimeException.class, () -> scanner.iterator().next().getKey());
```

Comment:
From this, it's not obvious which method is expected to throw. I think it throws on the call to `next()`, but am not sure. If so, then the `getKey()` method call should be dropped and the `scanner.iterator()` part be pulled out to a variable outside the `assertThrows`.